### ISSUE TITLE
<a-text> entity dont show correctly on Chrome and Chrome Canary for Android

### ISSUE THREAD

**Issue by darioodiaz** [aframevr/aframe#2670]
Dont show the text on Chrome in Android 4.4.2 (my phone) but also on iOS S6, on Firefox for Android works!

- A-Frame Version: 0.5
- Platform / Device: Chrome and Chrome Canary on Android 4.4.2
- Reproducible Code Snippet or URL:  

<a-scene stats>
     <a-entity position="0 6 4" rotation="0 180 0">
            <a-camera>               
                <a-text value="Hello" color="black" position="0 1.5 -3" scale="1.5 1.5 1.5"></a-text>
                <a-cursor></a-cursor>              
            </a-camera>
      </a-entity>
</a-scene>

**Comment by dmarcos** [comment]
Can you point to an example that reproduces the issue? Thanks

**Comment by darioodiaz** [comment]
https://s3.amazonaws.com/dariodiazbucket/chrome-issue.html

On chrome desktop works! on Chrome for Android the geometry-text works but the <a-text> dont

**Comment by machenmusik** [comment]
@darioodiaz can you provide more information on the specific model of phone that exhibits the problem?  my suspicion is that it does not support a OpenGLES extension the default font's text shader is using.  Can you also try one or more of the non-default stock fonts like kelsonsans?

**Comment by darioodiaz** [comment]
Yes, it's a LG G3 D693-AR which runs Android 4.4.2 with 3.4.67 kernel version, I have Chrome version 58. I test it with all the documentation stock fonts but it didn't work I'll test if that Chrome version or my Android version has OpenGLES and comment here, thanks for your time!

**Comment by machenmusik** [comment]
of specific interest: `GL_OES_standard_derivatives`

**Comment by bryik** [comment]
[Related?](https://github.com/bryik/aframe-bmfont-text-component/issues/28) The phones in both of these issues have similar GPUs (Mali-400). I thought the cause might be the use of highp precision in the SDF shader because [MDN says highp may not work on old phones](https://developer.mozilla.org/en-US/docs/Web/API/WebGL_API/WebGL_best_practices).

**Comment by dmarcos** [comment]
Any errors on console like in the lines of `extension XXX not supported`?

**Comment by jsantell** [comment]
On Chrome 58 on a Pixel, this is the result for me
![screenshot from 2017-05-19 10 49 31](https://cloud.githubusercontent.com/assets/641267/26260139/fcf2c338-3c80-11e7-80d5-5e7931ed13f3.png)

**Comment by darioodiaz** [comment]
@dmarcos 

THREE.WebGLRenderer: OES_texture_float extension not supported.
THREE.WebGLRenderer: OES_texture_float_linear extension not supported.
THREE.WebGLRenderer: OES_texture_half_float extension not supported.
THREE.WebGLRenderer: OES_texture_half_float_linear extension not supported.
THREE.WebGLRenderer: ANGLE_instanced_arrays extension not supported.
THREE.WebGLRenderer: OES_element_index_uint extension not supported.
THREE.WebGLRenderer: highp not supported, using mediump instead.
THREE.WebGLRenderer: image is not power of two (236x236). Resized to 256x256 <img crossorigin=​"anonymous" src=​"/​ball-texture.jpg">​

THREE.WebGLRenderer: EXT_texture_filter_anisotropic extension not supported.
THREE.WebGLShader: Shader couldn't compile.
THREE.WebGLShader: gl.getShaderInfoLog() fragment ERROR: 0:5: 'highp' : precision is not supported in fragment shader
THREE.WebGLProgram: shader error:  0 gl.VALIDATE_STATUS false gl.getProgramInfoLog invalid shaders  ERROR: 0:5: 'highp' : precision is not supported in fragment shader


this is the "cleaned" output from my Chrome for Android, as I see my phone doesnt support a WebGL feature so, a-text works but my phone doesnt support it, maybe a fallback if its possible to display text

**Comment by machenmusik** [comment]
If there is a good way to determine whether the shader should use highp or mediump this might be straightforward to fix, but I don't have anything to test it with
